core/pom.xml

```diff
@@ -135,6 +135,12 @@
       <version>${asciidoctorj-pdf.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.yaml</groupId>
+      <artifactId>snakeyaml</artifactId>
+      <version>1.15</version>
+    </dependency>
+
 
     <dependency>
       <groupId>org.jboss.arquillian.junit</groupId>
```

Comment:
Didnt say to remove your code, just to move it to support lazy (class)loading and make the dependency optional. Default needs to work even if user has conflicting libs (selenium will often be there and I d like to avoid "cukespace is bad" cause you need to play with libs by default). If a user uses the yaml config and didnt add the dependency, a logger.error with the dependency (copy/paste) to add is nice enough IMO.